pkgs/top-level/python-aliases.nix

```
@@ -395,6 +395,8 @@ mapAliases ({
   prometheus_client = prometheus-client; # added 2021-06-10
   prompt_toolkit = prompt-toolkit; # added 2021-07-22
   protonup = protonup-ng; # Added 2022-11-06
+  proton-vpn-network-manager-openvpn = throw "proton-vpn-network-manager-openvpn functionality was integrated in the proton-vpn-network-manager module"; # added 20-09-2024
+  proton-vpn-network-manager-wireguard = throw "proton-vpn-network-manager-wireguard functionality was integrated in the proton-vpn-network-manager module"; # added 20-09-2024
```

```suggestion
  proton-vpn-network-manager-openvpn = throw "proton-vpn-network-manager-openvpn functionality was integrated in the proton-vpn-network-manager module"; # added 2024-09-20
  proton-vpn-network-manager-wireguard = throw "proton-vpn-network-manager-wireguard functionality was integrated in the proton-vpn-network-manager module"; # added 2024-09-20
```
The date format should be YYYY-MM-DD to clean up aliases with a script.
https://github.com/NixOS/nixpkgs/blob/9c711566cde5929768e311413eaa2399631624ce/maintainers/scripts/remove-old-aliases.py#L56-L61

---

Done.